spec/e2e/pbivizPackageSpec.js

```diff
@@ -281,8 +281,211 @@ describe("E2E - pbiviz package", () => {
+            .then((pbivizJson) => {
+                expect(JSON.stringify(pbivizJson.stringResources) === JSON.stringify(validStringResources)).toBe(true);
+                done();
+            })
+            .catch((err) => {
+                expect(err).toBe(null);
+                done();
+            });
+    });
+
+    it("Should correctly generate pbiviz file with RESJSON and stringResources localizations", (done) => {
+        const resourceStringRuLocalization =
+            {
+                "locale": "ru-RU",
+                "values": {
+                    "formattingGeneral": "Главные настройки",
+                    "formattingGeneralOrientation": "Ориентация",
+                    "formattingHeaderFontColor": "Цвет шрифта",
+                    "formattingHeaderBackground": "Цвет фона"
+                }
+            };
+
+        const ResJsonEngLocalization =
+            {
+                "formattingGeneral": "General",
+                "formattingGeneralOrientation": "Orientation",
+                "formattingGeneralOrientationVertical": "Vertical",
+            };
+        const ResJsonRuLocalization =
+            {
+                "formattingGeneral": "Общие настройки",
+                "formattingGeneralOrientation": "Ориентация",
+                "formattingGeneralOrientationVertical": "Вертикальная"
+            };
+
+        const validStringResources =
+            {
+                "en-US": {
+                    "formattingGeneral": "General",
+                    "formattingGeneralOrientation": "Orientation",
+                    "formattingGeneralOrientationVertical": "Vertical"
+                },
+                "ru-RU": {
+                    "formattingGeneral": "Общие настройки",
+                    "formattingGeneralOrientation": "Ориентация",
+                    "formattingHeaderFontColor": "Цвет шрифта",
+                    "formattingHeaderBackground": "Цвет фона",
+                    "formattingGeneralOrientationVertical": "Вертикальная"
+                }
+            };
+
+        mkDirPromise('stringResources')
+            .then(() =>
+                Promise.all([
+                    mkDirPromise('stringResources/en-US')
+                        .then(() => writeJsonPromise('stringResources/en-US/resources.resjson', ResJsonEngLocalization)),
+                    mkDirPromise('stringResources/ru-RU')
+                        .then(() => writeJsonPromise('stringResources/ru-RU/resources.resjson', ResJsonRuLocalization)),
+                    writeJsonPromise('stringResources/ru-RU.json', resourceStringRuLocalization)
+                ]))
+            .then(() => readJsonPromise('pbiviz.json'))
+            .then((pbivizJson) => {
+                pbivizJson.stringResources = ["stringResources/ru-RU.json"];
+                return writeJsonPromise('pbiviz.json', pbivizJson);
+            })
+            .then(() => {
+                try {
```

Comment:
Are you sure that it has been fixed? I'm still able to find ```try``` statement. Why do we need it here?